README.md

```diff
@@ -22,7 +22,10 @@ This Bot Sample was tested with Mattermost server version 3.10.0.
 
 3 - Run `docker-compose up -d --build` and the mattermost client will be built and will expose the port `8065` to your system's localhost
 
-4 - Start the Bot.
+4 - Get a bot token by logging into your mattermost instance, copy `example.env` to `.env` and fill in the credentials. Alternatively, just provide your credentials as environment variables.
```

Comment:
(suggestion) Since there are a few steps to create a bot, maybe it makes sense to link to the directions page https://developers.mattermost.com/integrate/reference/bot-accounts/ e.g. `...logging into your mattermost instance and follow the instructions to create a bot...`

Reply:
Done